routes/users_routes.js

```diff
@@ -21,6 +21,321 @@ const router = express.Router();
 
 const { ensureAuth, csrfProtection, flashValidationErrors } = require('./middleware');
 
+const fetchLands = {
```

Comment:
Same comment on putting in a helper file.

Reply:
this is a temporary function for the database migration